PROCESS.md

```diff
@@ -0,0 +1,163 @@
+
+### STAGE-0
+`STAGE-0` features are tracked in the `STAGE-0` Registry on the website. This
+registry lists all `STAGE-0` features. Each entry SHOULD include the following
+checklist as well as an informal description with at least one example and any
+relevant links.
+
+This stage requires at least two implementations to implement the feature. The
+purpose of these implementations is to identify any potential implementation
+issues that might lead to low adoption such as difficulty of implementation or
+performance implications. Therefore, these implementations don't need to be
+publicly available or have real world use at this point.
+
+**Promotion Checklist:**
+[] There is general consensus among the core contributors for adding the feature
+  to the spec
+[] Tests are available in the test suite
+[] Two implementations have implemented the feature and pass the tests suite
+[] A blog post has been published introducing the feature
+[] A PR for the spec has been approved that completely specifies the feature and
+  removes the feature from the `STAGE-0` registry.
+
+### STAGE-1
+`STAGE-1` features are included to the specification, but flagged as `STAGE-1`
+and are not subject to the compatibility requirements of "stable" features. They
+can be added at any time as long as it meets all criteria for `STAGE-1` status.
+
+At this stage, the feature should be seeing real world use. These features MAY
+change or be removed all together based on what we learn from real world usage,
+but anything more than small changes should be extremely rare. Any necessary
+feature experimentation and evolution should have be done in `STAGE-0`.
+
+Implementers are encouraged to implement `STAGE-1` features, but are not
+expected to maintain support for previous versions if they change. Users who
+choose to use these features should be prepared for the possibility of changes
+and be willing to update their schemas if a change does occur.
+
+A `STAGE-1` feature can be promoted to `STAGE-2` at any time. Ideally a
+feature should stay in `STAGE-1` for about one year, but may stay longer if we
+don't see it used enough in the wild.
+
+**Promotion Checklist:**
+[] There is general consensus that the feature has been proven to be a good
+  addition to JSON Schema and is unlikely to change.
+[] We see the feature being used successfully in the wild and not generating a
+  lot of community support issues that could indicate a problem.
+
+### STAGE-2
+`STAGE-2` features are in the last stages of becoming stable. Changes are
+unlikely, but backward incompatible changes are still allowed in extreme cases.
+Users can be reasonably sure that the feature won't change, but can't expect all
+implementations to support the feature yet.
+
+`STAGE-2` is a staging area for features that are ready to be declared stable,
+but are waiting for the next release or don't yet have enough implementations.
+When we see the vast majority of actively maintained implementations support
```

Comment:
> the vast majority

Do we need to define this more rigidly?

Reply:
Maybe? Assuming bowtie grows to include most implementations, this decision could be made based on hard data rather than gut feel or a time consuming audit of all known implementations.